example/build_tools/bazel_rules/wheel_scanner/wheel_scanner.bzl

```diff
@@ -21,6 +18,14 @@ def _wheel_scanner_impl(target, ctx):
                     fail("Didn't have workspace prefix")
                 all_py_relative_paths.append(path[len(workspace_root) + 1:])
                 all_py_files.append(file)
+    elif ctx.rule.kind == "py_proto_library":
```

Comment:
The main changes are here. Instead of scanning for Python files in wheel, I'm grabbing the first file in DefaultInfo of the target.

Reply:
why not add all files and let the command sort it out?

Reply:
`DefaultInfo#files` returns all Python files including from the dependency, so import would end up adding a bunch of unnecessary deps.